src/main/flight/pid.c

```diff
@@ -1092,18 +1078,13 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
             // Log the value of D pre application of TPA
             preTpaD *= D_LPF_FILT_SCALE;
 
-            if (axis == FD_ROLL) {
-                DEBUG_SET(DEBUG_D_LPF, 2, lrintf(preTpaD));
-            } else if (axis == FD_PITCH) {
-                DEBUG_SET(DEBUG_D_LPF, 3, lrintf(preTpaD));
+            if (axis != FD_YAW) {
+                DEBUG_SET(DEBUG_D_LPF, axis - FD_ROLL + 2, lrintf(preTpaD));
```

Comment:
```suggestion
                DEBUG_SET(DEBUG_D_LPF, axis + 2, lrintf(preTpaD));
```

Reply:
This was added intentionally after discussion. if FD_ROLL changes from 0 it will still work.